plugins/builtin/apm/nomad/plugin/metrics.go

```diff
@@ -91,3 +95,47 @@ func (a *APMPlugin) Query(q string) (float64, error) {
 
 	return result, nil
 }
+
+func parseQuery(q string) (*Query, error) {
+	mainParts := strings.Split(q, "/")
+	if len(mainParts) != 3 {
```

Comment:
Since only jobs can have forward slashes I just switched the order of group and job and used `SplitN`. It's an internal representational, so changing it shouldn't be an issue.